src/Microsoft.DotNet.Interactive.Http.Tests/HttpKernelTests.cs

```diff
@@ -2233,6 +2233,51 @@ public async Task responses_to_named_requests_can_be_accessed_as_symbols_in_late
         secondResult.Events.Should().NotContainErrors();
     }
 
+    [Theory]
+    [InlineData("json.response.body.$.slideshow.slides.title")]
+    [InlineData("json.response.body.$.slideshow.slides.type")]
```

Comment:
Are we relying on httpbin.org to return a json that looks like this? I think it would be better to use a mock client with our own handler where we can provide our own json response (like we do in some other tests).

Otherwise, we could end up having to deal with failed builds in CI if httpbin.org happens to be down or if the format for the response ends up changing.

Reply:
It would also be nice to see what the response shape looks like in the test so that it is easier to understand what the test is validating...